src/pages/news-announcements.tsx

```diff
@@ -38,67 +39,75 @@ const NewsAnnouncements = ({
   return !user ? (
     <Loader />
   ) : (
-    <div className={styles.listContainer}>
-      {announcements.length > 0 && (
-        <section>
-          <Announcement announcements={announcements} />
-        </section>
-      )}
+    <>
+      <Head>
+        <title>News & Annoucements - USSF Portal</title>
+      </Head>
```

Comment:
Suggestion: can this page use the `pageTitle` or is there some reason it needs to use the `Head` tag?